Update the `load` command description as it supports full backups (#1594)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/restore-dump.adoc

A database dump can be loaded to a Neo4j instance using the `load` command of `neo4j-admin`.

> Starting with Neo4j 5.20, `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].

Author: ```suggestion
Starting with Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].
```

> Starting from Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].

Author: ```suggestion
Starting from Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].

You can use this functionality for restoring full backup files created in the Aura instance (link:https://neo4j.com/docs/aura/auradb/managing-databases/backup-restore-export/).
```

> Starting from Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].

Reviewer: ```suggestion
From Neo4j 5.20, the `neo4j-admin database load` command also supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].
```

[NOTE]
====
Change Data Capture does **not** capture any data changes resulting from the use of `neo4j-admin database load`.

> See link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#non-tx-log-changes[Change Data Capture -> Points to consider] for more information.

Author: ```suggestion
See link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#non-tx-log-changes[Change Data Capture -> Key considerations] for more information.
```

====

[[restore-dump-command]]
== Command

> The `neo4j-admin database load` command loads a database from an archive created with the xref:backup-restore/offline-backup.adoc#offline-backup-command[`neo4j-admin database dump`] command or from xref:backup-restore/online-backup.adoc#online-backup-command[full Neo4j Enterprise backup] starting with Neo4j 5.20.

Reviewer: ```suggestion
The `neo4j-admin database load` command loads a database from an archive created with the xref:backup-restore/offline-backup.adoc#offline-backup-command[`neo4j-admin database dump`] command or from xref:backup-restore/online-backup.adoc#online-backup-command[full Neo4j Enterprise backup] (available from Neo4j 5.20).
```

=== Description

Load a database from an archive.
_<archive-path>_ must be a directory containing an archive(s).
Archive can be a database dump created with the dump command, or can be a full backup artifact created by the backup command from Neo4j Enterprise.

> If neither --from-path or --from-stdin is supplied `server.directories.dumps.root` setting will be searched for the archive.

Reviewer: ```suggestion
If neither `--from-path` or `--from-stdin` is supplied `server.directories.dumps.root` setting will be searched for the archive.
```

> Existing databases can be replaced by specifying --overwrite-destination.

Reviewer: ```suggestion
Existing databases can be replaced by specifying `--overwrite-destination`.
```

It is not possible to replace a database that is mounted in a running Neo4j server.

> If --info is specified, then the database is not loaded, but information (i.e. file count, byte count, and format of load file) about the archive is printed instead.

Reviewer: ```suggestion
If `--info` is specified, then the database is not loaded, but information (i.e. file count, byte count, and format of load file) about the archive is printed instead.
```

bin/neo4j-admin database load --from-path=s3://full/path/to/aws-s3-dumps/ <database> --overwrite-destination=true
----

> The following example shows how to specify a specific file for the `load` command, as of Neo4j 5.20.

Author: ```suggestion
The following example shows how to designate a specific archive for the `load` command, as of Neo4j 5.20.
```

> The following example shows how to designate a specific archive for the `load` command, as of Neo4j 5.20.

Reviewer 2: remove `as of Neo4j 5.20` this functionality is there since 4.something

bin/neo4j-admin database load --from-path=/full-path/data/dumps <database> --overwrite-destination=true
----

Starting from Neo4j 5.20, you can use the same command to load the database from its full backup artifact:

[source,shell, role="nocopy"]
----

> bin/neo4j-admin database load --from-path=/full-path/to/backups <database> --overwrite-destination=true

Reviewer: I wonder if the path shouldn't point to the artifact as it does in the `neo4j-admin database restore` command.

Author: tbh, I think it's redundant in this case
